Tests/HyperwalletPaypalAccountTests.swift

```diff
@@ -0,0 +1,281 @@
+        paypalAccountResponse = result
+        errorResponse = error
+        expectation.fulfill()
+        })
+        wait(for: [expectation], timeout: 1)
+
+        // Then
+        XCTAssertNotNil(errorResponse, "The `errorResponse` should not be nil")
+        XCTAssertNil(paypalAccountResponse)
+        XCTAssertEqual(errorResponse?.getHttpCode(), 400)
+        XCTAssertEqual(errorResponse?.getHyperwalletErrors()?.errorList?.first?.fieldName, "email")
+    }
+
+    func testGetPaypalAccount_success() {
+        // Given
+        let expectation = self.expectation(description: "Get paypal account completed")
+        let response = HyperwalletTestHelper.okHTTPResponse(for: "PaypalAccountResponse")
+        let url = String(format: "%@/paypal-accounts/trm-12345", HyperwalletTestHelper.userRestURL)
+        let request = HyperwalletTestHelper.buildGetRequest(baseUrl: url, response)
+        HyperwalletTestHelper.setUpMockServer(request: request)
+
+        var paypalAccountResponse: HyperwalletPaypalAccount?
+        var errorResponse: HyperwalletErrorType?
+
+        // When
+        Hyperwallet.shared.getPaypalAccount(transferMethodToken: "trm-12345", completion: { (result, error) in
+            paypalAccountResponse = result
+            errorResponse = error
+            expectation.fulfill()
+        })
+        wait(for: [expectation], timeout: 1)
+
+        // Then
+        XCTAssertNil(errorResponse, "The `errorResponse` should be nil")
+        XCTAssertNotNil(paypalAccountResponse?.getFields())
+        XCTAssertEqual(paypalAccountResponse?.getField(fieldName: .email) as? String, "test@paypal.com")
+    }
+
+    func testUpdatePaypalAccount_success() {
+        // Given
+        let expectation = self.expectation(description: "Update paypal account completed")
+        let response = HyperwalletTestHelper.okHTTPResponse(for: "PaypalAccountResponse")
+        let url = String(format: "%@/paypal-accounts/trm-12345", HyperwalletTestHelper.userRestURL)
+        let request = HyperwalletTestHelper.buildPutRequest(baseUrl: url, response)
+        HyperwalletTestHelper.setUpMockServer(request: request)
+
+        var paypalAccountResponse: HyperwalletPaypalAccount?
+        var errorResponse: HyperwalletErrorType?
+
+        // When
+        let paypalAccount = HyperwalletPaypalAccount.Builder(token: "trm-12345").email("test@paypal.com").build()
+
+        Hyperwallet.shared.updatePaypalAccount(account: paypalAccount, completion: { (result, error) in
+            paypalAccountResponse = result
+            errorResponse = error
+            expectation.fulfill()
+        })
+        wait(for: [expectation], timeout: 1)
+
+        // Then
+        XCTAssertNil(errorResponse, "The `errorResponse` should be nil")
+        XCTAssertNotNil(paypalAccountResponse?.getFields())
+        XCTAssertEqual(paypalAccountResponse?.getField(fieldName: .email) as! String, "test@paypal.com")
+    }
+
+    func testUpdatePaypalAccount_invalidEmail() {
```

Comment:
Change this test case to handle this response 
``` {
    "errors": [
        {
            "message": "PayPal transfer method email address should be same as profile email address.",
            "code": "CONSTRAINT_VIOLATIONS"
        }
    ]
}
```

Reply:
Added another test for this case